### ISSUE TITLE
App Not Working

### ISSUE THREAD

**Issue by kianrafiee** [BigBerny/magicquit#11]
I installed the app and opened it but there is no menu or settings or anything. Just an active app in Activity Monitor. Same goes for other quit applications like SwiftQuit and Quitter.

Mac OS Sequoia  15.0
MacBook Pro M3

**Comment by giraudvalentin** [comment]
@kianrafiee i have the same problem. You close this issue, did you resolve it and how ?

**Comment by kianrafiee** [comment]
@giraudvalentin I never resolved the issue. I ended up using SwiftQuit instead.

**Comment by BigBerny** [comment]
It should simply appear in the Menu Bar close to the clock. I use it on Sequoia without a problem.

**Comment by BigBerny** [comment]
@kianrafiee SwiftQuit is nice but doing something completely different.

**Comment by kianrafiee** [comment]
> It should simply appear in the Menu Bar close to the clock. I use it on Sequoia without a problem.

Nope. Not for me. Just installed it again to see if the problem persists and it's the same thing again. No menu option near the clock, just an active process that does nothing until I force quit it.

**Comment by giraudvalentin** [comment]
I had to restart my mac then it work know... strange

**Comment by francoisauclair911** [comment]
Same issue here, it disappears  from the menu bar after a while. And then if I forcequit and restart the app it still does not appear in the menu bar

**Comment by MickG-man** [comment]
Yeah, it really doesn't seem to be launched, but BigBerny's tip proved true for me.  Interesting, because the app itself in the dock appears unlaunched.

Next I'll see if it actually does quit ARD, which now essentially disables the Macs it takes over using and the only workaround is to return to the Mac it was launched on, Quit ARD, and then you can regain control of the other mac it has taken over.

**Comment by MickG-man** [comment]
Yes, it works! I didn't realize BigBerny was the creator and just wanted to thank him for this.  It does exactly what I need it to. 

Thanks, man!
